source-web/src/assets/db/tools.json

```diff
@@ -126,5 +126,23 @@
 		"img": "https://res.cloudinary.com/cosmocloudinary/image/upload/v1707069166/freesets/tools/hvxjb3ktzko8xztb1qcl.webp",
 		"link": "https://transform.tools/",
 		"tags": ["Transform data"]
+	},
+	{
+		"id": 17,
+		"order": 30,
+		"name": "Favicon Generator",
+		"img": "https://res.cloudinary.com/cosmocloudinary/image/upload/freesets/icons/favicon-generator",
```

Comment:
The correct url would be: https://res.cloudinary.com/cosmocloudinary/image/upload/freesets/tools/favicon-generator